crates/sui-rpc-api/proto/sui/rpc/v2alpha/balance_change.proto

```diff
@@ -0,0 +1,18 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+syntax = "proto3";
+
+package sui.rpc.v2alpha;
+
+// The delta, or change, in balance for an address for a particular `Coin` type.
+message BalanceChange {
+  // The account address that is affected by this balance change event.
+  optional string address = 1;
+
+  // The `Coin` type of this balance change event.
+  optional string coin_type = 2;
+
+  // The amount or change in balance.
+  optional string amount = 3;
```

Comment:
Just curious, why `string` here? Not an `int64`?

Reply:
because this needs to represent an i128 as the max value you can have for a coin is u64 so a delta or change of u64::MAX wouldn't fit properly in an int64